### DISCUSSION TITLE
Reinstatement Request, Response Time

### DISCUSSION THREAD

**Discussion by Tyraelgallen** [community/community#116661] [New to GitHub]
Hello everybody.
i wanted to ask if anyone know how long it took (to you) to get a response to your Reinstatement Request from GITHUB Support.
For me its been nearly 2 months. My account was flagged as spammy (i have an organization account- not ended in @gmail.com )

i saw a recent post that @GoodbyeKittyy  wrote about the same issue & @queenofcorgis  solved.
see discussion here: [https://github.com/orgs/community/discussions/113684](url)
flagged account: [https://github.com/emilio-borovski-ueno](url)

What can i do to improve or reduce my response/solve time?

if any support engineer sees this, my personal tickets is #2710380
--------------------------------------------------------------------------------
EDIT:
i had to make this other discussion because my original ticket/discussion was invisible due to my account being flagged [https://github.com/emilio-borovski-ueno](url)
i find it odd that you cannot ask for a reinstatement request from the flagged account & only from another (non-flagged) accounts

**Comment by ghostinhershell** [comment]
Hi @Tyraelgallen,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
